insights/parsers/systemd/unitfiles.py

```diff
@@ -166,3 +166,47 @@ def get_service_details(self, service_name):
         """
         empty_details = {'LOAD': None, 'ACTIVE': None, 'SUB': None, 'UNIT': None}
         return self.unit_list.get(service_name, empty_details)
+
+    def is_loaded(self, service_name):
+        """
+        Return the LOAD state of service managed by systemd.
+
+        Args:
+            service_name (str): service name including its extension.
+
+        Returns:
+            bool: True if service is loaded False if not loaded
+        """
+        empty_details = {'LOAD': None, 'ACTIVE': None, 'SUB': None, 'UNIT': None}
```

Comment:
Since you already have more or less this logic in get_service_details, you should be able to conflate these three lines to:
```
return self.get_service_details(service_name)['LOAD'] == 'loaded'
```